setup.py

```diff
@@ -1,68 +0,0 @@
-from setuptools import setup, find_packages
```

Comment:
Would it be possible to only add support of pybuilder, without removing support of setuptools & co?

Reply:
Everything is possible, but why support 2 build chains? We can do this for the moment of course.